integration-test/Css.ts

```diff
@@ -119,25 +119,25 @@ class CssBuilder<T extends Properties1> {
   display(value: Properties["display"]) { return this.add("display", value); }
 
   // flexbox
-  get justifyStart() { return this.add("justifyContent", "flex-start"); }
-  get justifyEnd() { return this.add("justifyContent", "flex-end"); }
-  get justifyCenter() { return this.add("justifyContent", "center"); }
-  get justifyBetween() { return this.add("justifyContent", "space-between"); }
-  get justifyAround() { return this.add("justifyContent", "space-around"); }
-  get justifyEvenly() { return this.add("justifyContent", "space-evenly"); }
-  justify(value: Properties["justifyContent"]) { return this.add("justifyContent", value); }
-  get selfStart() { return this.add("alignSelf", "flex-start"); }
-  get selfEnd() { return this.add("alignSelf", "flex-end"); }
-  get selfCenter() { return this.add("alignSelf", "center"); }
-  get selfBaseline() { return this.add("alignSelf", "baseline"); }
-  get selfStretch() { return this.add("alignSelf", "stretch"); }
-  self(value: Properties["alignSelf"]) { return this.add("alignSelf", value); }
-  get itemsStart() { return this.add("alignItems", "flex-start"); }
-  get itemsEnd() { return this.add("alignItems", "flex-end"); }
-  get itemsCenter() { return this.add("alignItems", "center"); }
-  get itemsBaseline() { return this.add("alignItems", "baseline"); }
-  get itemsStretch() { return this.add("alignItems", "stretch"); }
-  items(value: Properties["alignItems"]) { return this.add("alignItems", value); }
+  get jcfs() { return this.add("justifyContent", "flex-start"); }
+  get jcfe() { return this.add("justifyContent", "flex-end"); }
+  get jcc() { return this.add("justifyContent", "center"); }
+  get jcsb() { return this.add("justifyContent", "space-between"); }
+  get jcsa() { return this.add("justifyContent", "space-around"); }
+  get jce() { return this.add("justifyContent", "space-evenly"); }
```

Comment:
what are your thoughts on making this `jcse` to be consistent with `jcsb` and `jcsa`?

Reply:
Oh yes! I missed that, will change...